twib/Logger.hpp

```diff
@@ -1,6 +1,51 @@
 #pragma once
 
 #include<memory>
+#include<ostream>
+
+namespace neolib
```

Comment:
Is this actually used anywhere?
It's still a useful thing to have around, but I think it belongs in a different header file.

Reply:
Ah no it's not used anywhere, I just copy pasted this and kept it around because it was useful :P